spring-boot/src/main/resources/hotrod-client.properties

```diff
@@ -0,0 +1 @@
+infinispan.client.hotrod.server_list=127.0.0.1:11222
```

Comment:
This property file is read by a `RemoteCacheManager`'s configuration [here](https://github.com/infinispan/infinispan-spring-boot/pull/7/files#diff-633ddf90ddf8ca6ccee9e06c7bc964ceR47). It is written in our format, please have a look into [the documentation](http://infinispan.org/docs/stable/user_guide/user_guide.html#configuration_9).

Hot Rod Client is used in Client/Server mode, where a client connects to a remote Server.